client-spark/spark3/src/main/java/org/apache/spark/shuffle/DelegationRssShuffleManager.java

```diff
@@ -104,6 +104,7 @@ private ShuffleManager createShuffleManagerInDriver() throws RssException {
               Constants.SORT_SHUFFLE_MANAGER_NAME, sparkConf, true);
       sparkConf.set(RssSparkConfig.RSS_ENABLED.key(), "false");
       sparkConf.set("spark.shuffle.manager", "sort");
+      sparkConf.set("spark.shuffle.service.enabled", "true");
```

Comment:
If the user doesn't enable external shuffle, this config option shouldn't be `true`.

Reply:
But if it is `false`, `sort` shuffle manager cannot work

Reply:
sort shuffle manager can work without external shuffle.